Gems/EMotionFX/Code/Source/Integration/Components/SimpleMotionComponent.cpp

```diff
@@ -307,6 +307,12 @@ namespace EMotionFX
         {
             if (m_motionInstance)
             {
+                if (time > m_motionInstance->GetDuration())
+                {
+                    AZ_Warning("EMotionFX", false, "Invalid play time: %f, The set value exceeds the maximum value.", time);
+                    return;
+                }
```

Comment:
What about printing the warning here but also just clamp the value to `time = m_motionInstance->GetDuration();` rather than returning entirely?